cmake/extensions.cmake

```diff
@@ -889,6 +889,12 @@ function(import_kconfig prefix kconfig_fragment)
   endforeach()
 endfunction()
 
+macro(get_image_name image out_var)
```

Comment:
Function's should be used whenever possible, AFAICT it is possible here.

(less chance of aliasing issues due to the creation of a new scope)

Reply:
Will fix